### DISCUSSION TITLE
Notifications only for selected component

### DISCUSSION THREAD

**Discussion by ALIENQuake** [WeblateOrg/weblate#6067] [Q&A] [answered]
Hello!

I could not find this on the documentation page:

Does weblate allow for notifications only for the selected component, not the entire project?

**Comment by nijel** [comment, marked as answer]
Yes, it does. You can adjust this from component page, see https://docs.weblate.org/en/latest/user/profile.html#notifications

**Reply by ALIENQuake** [reply]
I was reading 3.1.1 docs, thanks!

**Reply by nijel** [reply]
That version indeed didn't support that...
